src/js/cordova_chromeapi.js

```diff
@@ -60,6 +60,13 @@ const chromeapiSerial = {
         stopBits: 'one',
         ctsFlowControl: false,
     },
+    getDriver: function(vid, pid) {
+        if (vid === 4292 && pid === 60000) {
+            return 'Cp21xxSerialDriver'; //for Silabs CP2102 and all other CP210x
+        }  else {
+            return undefined;
```

Comment:
maybe assign the default here:
```suggestion
            return 'CdcAcmSerialDriver';
```

Reply:
@danybd Did you also test your patch if the connection with a `VCP` device is still working?

Reply:
> @danybd Did you also test your patch if the connection with a `VCP` device is still working?

yes I did. I tested both of them.